tests/DuskTestCase.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Tests;
+
+use Rogue\Models\User;
+use Laravel\Dusk\TestCase as BaseTestCase;
+use Facebook\WebDriver\Chrome\ChromeOptions;
+use Facebook\WebDriver\Remote\RemoteWebDriver;
+use Facebook\WebDriver\Remote\DesiredCapabilities;
+
+abstract class DuskTestCase extends BaseTestCase
+{
+    use CreatesApplication, WithMocks, WithAuthentication;
+
+    /**
+     * The Faker generator, for creating test data.
+     *
+     * @var \Faker\Generator
+     */
+    protected $faker;
+
+    /**
+     * Get the default user for Dusk's login() helper.
+     *
+     * @return User
+     */
+    protected function user()
+    {
+        return factory(User::class, 'admin')->create();
+    }
+
+    /**
+     * Setup the test environment.
+     *
+     * @return void
+     */
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->configureMocks();
```

Comment:
Are we able to use certain types of mocks in Dusk? Is this needed?

Reply:
Oooof, yeah noooope! This was from my earlier more naïve days.

Reply:
😆